### PR TITLE
Fixing dependencies.

### PR DESCRIPTION
**Pull request by 0xdeafbeef**

### CODE REVIEW

**Comment by 0xdeafbeef** [comment]
Maybe we need fix ssh2 on some commit

**Comment by spebern** [comment]
Thanks for taking a look at this!

I recently pushed some changes to `ssh2-rs` and they should be available with version `0.7`.  For now `async-ssh2` is still a work in progress.

I cannot really reproduce the build issues that you have. Are you using the crate from within another crate, or did you use cargo install?


> Maybe we need fix ssh2 on some commit
What needs to be fixed?

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ edition = "2018"
 vendored-openssl = ["ssh2/vendored-openssl"]
 
 [dependencies]
-ssh2 = "0.7"
+ssh2 = {git = "https://github.com/alexcrichton/ssh2-rs", branch = "send_safe" }
```
Comment (spebern) [inline Cargo.toml:18]: I think adding the dependency on the branch here would make the [patch.crates-io] section irrelevant.

**Comment by 0xdeafbeef** [comment]
Building  lib standalone works, but using lib inside my project doesn't work - build fails. Soon I'll open sources of my project

**Pull request closed without merging**